packages/canary-integration/src/__tests__/capability.test.ts

```diff
@@ -169,159 +191,160 @@ describe('CACAO Integration test', () => {
-            asDID: didKeyWithCapability,
-            anchor: false,
-            publish: false,
-          })
-        ).rejects.toThrow(/invalid_jws/)
-      }, 30000)
-
-      test('can update stream with family resource', async () => {
-        const family = 'testFamily1'
-        // Create a determinstic tiledocument owned by the user
-        const deterministicDocument = await TileDocument.deterministic(ceramic, {
-          deterministic: true,
-          family,
-          controllers: [`did:pkh:eip155:1:${wallet.address}`],
+    test('fails to create if cacao issuer is not document controller using model resource', async () => {
+      const didKeyWithCapability = await addCapToDid(
+        wallet,
+        didKey2,
+        `ceramic://*?model=${METADATA.model.toString()}`
+      )
+      ceramic.did = didKeyWithCapability
+
+      await expect(
+        ModelInstanceDocument.create(ceramic, CONTENT0, {
+          model: FAKE_STREAM_ID,
+          controller: `did:key:z6MkwDAbu8iqPb2BbMs7jnGGErEu4U5zFYkVxWPb4zSAcg39#z6MkwDAbu8iqPb2BbMs7jnGGErEu4U5zFYkVxWPb4zSAcg39`,
         })
-        const streamId = deterministicDocument.id
-        const didKeyWithCapability = await addCapToDid(
-          wallet,
-          didKey,
-          `ceramic://*?family=${family}`
-        )
+      ).rejects.toThrow(/invalid_jws/)
+    }, 30000)
 
-        await deterministicDocument.update({ foo: 'bar' }, null, {
-          asDID: didKeyWithCapability,
+    test('fails to update if cacao issuer is not document controller using model resource', async () => {
+      const didKeyWithCapability = await addCapToDid(
+        wallet,
+        didKey,
+        `ceramic://*?model=${METADATA.model.toString()}`
+      )
+
+      ceramic.did = didKeyWithCapability
+
+      const doc = await ModelInstanceDocument.create(ceramic, CONTENT0, {
+        model: FAKE_STREAM_ID,
+        controller: `did:pkh:eip155:1:${wallet.address}`,
+      })
+
+      const didKeyWithBadCapability = await addCapToDid(
+        wallet2,
+        didKey2,
+        `ceramic://*?model=${METADATA.model.toString()}`
+      )
+
+      await expect(
+        doc.replace(CONTENT1, {
+          asDID: didKeyWithBadCapability,
           anchor: false,
           publish: false,
         })
+      ).rejects.toThrow(/Failed/)
+    }, 30000)
 
-        expect(deterministicDocument.content).toEqual({ foo: 'bar' })
-      }, 30000)
-    })
 
-    describe('Create stream', () => {
-      test('can not create new stream with wrong family', async () => {
-        const family = 'testFamily1'
-        const didKeyWithCapability = await addCapToDid(
-          wallet,
-          didKey,
-          `ceramic://*?family=${family}`
-        )
+    test('can create stream with model resource', async () => {
+      const didKeyWithCapability = await addCapToDid(
+        wallet,
+        didKey,
+        `ceramic://*?model=${METADATA.model.toString()}`
```

Comment:
can you either use `FAKE_STREAM_ID` here or use `METADATA.model` below when making the stream?  It's confusing to see two different things in the test and have to realize that they are actually the same.

Ditto for the other tests

Reply:
yeah agree, added more consistent naming